NodeGraphSwift-iosTests/AbstractNodeTests.swift

```diff
@@ -151,19 +151,18 @@ class AbstractNodeTests: XCTestCase {
         var i = 0
         let start = Date().timeIntervalSince1970
         var testTime: TimeInterval = 0.0
-        var iterator: (() -> Void)? = nil
-        iterator = {
-            i += 1
+        
+        var dispatchGroup = DispatchGroup()
```

Comment:
It appears this fixes the issue we were having:
![Screenshot 2019-09-19 at 17 03 09](https://user-images.githubusercontent.com/30214449/65256300-9b634280-daff-11e9-8807-50ecb0c01c66.png)
![Screenshot 2019-09-19 at 17 02 49](https://user-images.githubusercontent.com/30214449/65256307-a0c08d00-daff-11e9-829c-353d0e1fce87.png)